app/models/zone.rb

```diff
@@ -68,6 +68,7 @@ def self.create_maintenance_zone
 
       # 2) Assign to MiqRegion
       MiqRegion.my_region.update(:maintenance_zone => zone)
+      Module.clear_all_cache_with_timeout
```

Comment:
I had a test fail because the cache had been hit before this record was created.

Reply:
We should clear every cache everywhere though, right?  We should be selective in this code.  Do you know which cache was causing an issue?

Reply:
I'm assuming it's the `cache_with_timeout(:maintenance_zone)`, so we can just clear that specific cache with.

```ruby
Zone.maintenance_zone_clear_cache
```

Reply:
Actually - I can't understand why we are caching that at all.  The contents of the method is 

```
  # Zone for paused providers (no servers in it), not visible by default
  cache_with_timeout(:maintenance_zone) do
    MiqRegion.my_region&.maintenance_zone
  end
```

but `MiqRegion.my_region` itself is cached with timeout.  This is pulling a cached value out of an already cached value, which seems pointless.  When the cache is over to my_region, then it will get picked up anyway.